packages/api/lib/api-gateway.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const { Cookie } = require('tough-cookie');
+
+/**
+ * Normalize the headers in an API Gateway Lambda Proxy request or response
+ *
+ * From the AWS docs:
+ *
+ *   - The headers key can only contain single-value headers.
+ *   - The multiValueHeaders key can contain multi-value headers as well as
+ *     single-value headers.
+ *   - If you specify values for both headers and multiValueHeaders, API Gateway
+ *     merges them into a single list. If the same key-value pair is specified
+ *     in both, only the values from multiValueHeaders will appear in the merged
+ *     list.
+ * @param {Object} request - an API Gateway Lambda Proxy request or response
+ * @returns {Object} a mapping of lower-case header names to arrays of header
+ *   values
+ */
+function normalizeHeaders(request) {
+  const headers = {};
+
+  Object.entries(request.headers || {}).forEach(([key, value]) => {
+    headers[key.toLowerCase()] = [value];
+  });
+
+  Object.entries(request.multiValueHeaders || {}).forEach(([key, values]) => {
```

Comment:
I don't understand this. If the AWS docs say that it will handle `headers` and `multiValueHeaders` for us, then why are we merging them together here?

Reply:
Yeah, I think that I misunderstood what they were trying to say in the documentation.  I've removed the `normalizeHeaders` function.